official/nlp/data/electra_create_pretraining_data.py

```diff
@@ -0,0 +1,462 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
The argument list of create_training_instances changed, but when called, masked_lm_prob is still used.
Also, function create_masked_lm_predictions is not used, you can remove it.
I'm thinking that to minimize code change and potential errors, you can modify data/create_pretraining_data.py: when masked_lm_positions=None and masked_lm_labels=None, it generates electra data.